project/Environment.scala

```diff
@@ -21,7 +21,7 @@ object Environment {
     Properties.envOrElse(environmentVariable, default)
 
   lazy val hadoopVersion  = either("SPARK_HADOOP_VERSION", "2.2.0")
-  lazy val sparkVersion   = either("SPARK_VERSION", "1.4.1")
+  lazy val sparkVersion   = either("SPARK_VERSION", "1.5.0")
```

Comment:
Both EMR and CDH depend on 1.5.0, this can be changed later if it needs to be...I'm actually compiling against 1.5.2 in the client app